subjects/devops/auto-jobs/README.md

```diff
@@ -0,0 +1,70 @@
+## auto-jobs
+
+### Instructions
+
+In this exercise you will create different files `task1`, `task2`, `task3`, `task4` containing the formulas required for setup a group of  Scheduled tasks in the group.
+
+##### task1:
+Time: `Every Friday, at 20:41`
+Command: `echo "01"`
+
+##### task2:
+Time: `Every minute`
+Command: `sh /home/user01/check`
+
+##### task3:
+Time: `Everyday midnight at 12 AM`
```

Comment:
For this task you solution is returning this:

` 1 0 * * * sh /home/user01/backup$`
At 00:01

i think it should be this:
 `0 0 * * * sh /home/user01/backup$`
At 00:00 
Right?

Reply:
yes correct, fixed!